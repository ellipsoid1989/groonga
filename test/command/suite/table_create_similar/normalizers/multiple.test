table_create Normalizations TABLE_PAT_KEY ShortText
column_create Normalizations normalized COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizers 'NormalizerNFKC130("unify_kana", true), \
                 NormalizerTable("normalized", "Normalizations.normalized")'

table_create_similar TermsSimilar Terms

dump
