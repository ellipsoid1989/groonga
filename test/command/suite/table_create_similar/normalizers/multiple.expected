table_create Normalizations TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Normalizations normalized COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizers 'NormalizerNFKC130("unify_kana", true),                  NormalizerTable("normalized", "Normalizations.normalized")'
[[0,0.0,0.0],true]
table_create_similar TermsSimilar Terms
[[0,0.0,0.0],true]
dump
table_create Normalizations TABLE_PAT_KEY ShortText
column_create Normalizations normalized COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizers "NormalizerNFKC130(\"unify_kana\", true), NormalizerTable(\"normalized\", \"Normalizations.normalized\")"

table_create TermsSimilar TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizers "NormalizerNFKC130(\"unify_kana\", true), NormalizerTable(\"normalized\", \"Normalizations.normalized\")"
